yamlExamples/3/example.yaml

```diff
@@ -0,0 +1,51 @@
+# same as defined in current version
+agent:
+  label: my_label
+  customWorkspace: path_to_workspace
+  dockerImage: maven:3-alpine
+  args: -v /tmp:/tmp
+  # so that tools will be a prt of agent only
+  # tools needs to be defined in the tool configuration
+  tools:
+    maven : maven_3.0.1
+    jdk : jdk8
+
+# if true PR will be fetched and build will be started without merging it to the target branch
+# else PR will be merged to the target branch and then build will be started.
+# default value will be false
+buildPROnly: true
+
+# Push PR changes to the target branch if the build will be success.
+# default value will be false
+pushPROnSuccess: true
+
+# Location of reports to be published
+reports:
+  - location_of_report_1
```

Comment:
this comment was not addressed. I still do not understand what is supposed to do

Reply:
Will be addressed before the final release as disscussed in PR #16.